shipyard_blueprints/box/shipyard_box/cli/download.py

```diff
@@ -43,24 +43,12 @@ def get_args():
     return parser.parse_args()
 
 
-def set_environment_variables(args):
-    """
-    Set Box Service Account Credentials as environment variables if they're provided via keyword
-    arguments rather than seeded as environment variables. This will override
-    system defaults.
-    """
-    if args.service_account:
-        os.environ["BOX_APPLICATION_CREDENTIALS"] = args.service_account
-    return
-
-
 def extract_file_name_from_source_full_path(source_full_path):
     """
     Use the file name provided in the source_file_name variable. Should be run only
     if a destination_file_name is not provided.
     """
-    destination_file_name = os.path.basename(source_full_path)
-    return destination_file_name
+    return os.path.basename(source_full_path)
```

Comment:
just cleaned up a bit- no need for the return value to be stored as a variable if it is just going to be immediately returned